h2o-py/demos/uplift_drf_demo.ipynb

```diff
@@ -0,0 +1,2844 @@
+      "text/plain": [
+       "<Figure size 720x432 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plot_uplift_score(h2o_uplift_pred['uplift_predict'].as_data_frame())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Evaluate the model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 19,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "perf_h2o = h2o_uplift_model.model_performance(h2o_test_df)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "\n",
+    "\n",
+    "### Area Under Uplift Curve (AUUC) calculation\n",
+    "\n",
+    "To calculate AUUC for big data, the predictions are binned to histograms. Due to this feature the results should be different compared to exact computation.\n",
+    "\n",
+    "To define AUUC, binned predictions are sorted from largest to smallest value. For every group the cumulative sum of observations statistic is calculated. The uplift is defined based on these statistics.\n",
+    "\n",
+    "\n",
+    "#### Types of AUUC\n",
+    "\n",
+    "\n",
+    "| AUUC type | &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;&nbsp; &nbsp; &nbsp; Formula &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp;|\n",
+    "|:----------:|:-------------------------------------------:|\n",
+    "| **Qini**   | $TY1 - CY1 * \\frac{T}{C}$                   |\n",
+    "| **Lift**   | $\\frac{TY1}{T} - \\frac{CY1}{C}$             |\n",
+    "| **Gain**   | $(\\frac{TY1}{T} - \\frac{CY1}{C}) * (T + C)$ |\n",
+    "\n",
+    "\n",
+    "Where:\n",
+    "\n",
+    "- **T** how many observations are in the treatment group (how many data rows in the bin have treatment_column label == 1)\n",
+    "- **C** how many observations are in the control group (how many data rows in the bin have treatment_column label == 0)\n",
+    "- **TY1** how many observations are in the treatment group and respond to the offer (how many data rows in the bin have treatment_column label == 1 and response_column label == 1)\n",
+    "- **CY1** how many observations are in the control group and respond to the offer (how many data rows in the bin have treatment_column label == 0 and response_column label == 1)\n",
+    "\n",
+    "\n",
+    "The resulting AUUC value is not normalized, so the result could be a positive number, but also a negative number. A higher number means better model. \n",
```

Comment:
They are normalized now

Reply:
Here I would be more specific. I didn't get that you are talking about numbers comes from Formula up there but I connect it to the table below where is some normalized AUUC.

Can we say something like:

The resulting AUUC value is:

-  Not normalized.
- The result could be a positive or negative number. 
- Higher number means better model. 

More information about normalization is in **Normalized AUUC** section.